include/dp_port.h

```diff
@@ -39,6 +39,7 @@ struct dp_port_iface {
 	uint8_t					ul_ipv6[DP_IPV6_ADDR_SIZE];
 	uint32_t				nat_ip;
 	uint16_t				nat_port_range[2];
+	bool					is_ipv6_set;
```

Comment:
I would suggest `ipv6_set`, since `is_xxx` usually means the whole structure, like `port->is_attached`. But that may be my non-english-speaking background.

Reply:
I was also hesitant what to choose, ipv6_set sounds a bit better though after giving it a second thought.